AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java

```diff
@@ -0,0 +1,74 @@
+package Ex2Upgrade;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.support.How;
+
+public class Ex2UpgradeMain {
+    public static WebDriver driver;
+    public static void main (String []args) throws InterruptedException {
+
+        openBrowser("Chrome");
+        visitWebsite("https://the-internet.herokuapp.com/login");
+
+        fill(getElement(How.NAME,"username"),"tomsmith");
+        fill(getElement(How.NAME,"password"),"SuperSecretPassword!");
```

Comment:
You can call getElement inside fill method and just input 3 variables ( how, locator, inputText). 
And your main will be more similar to a test case ( almost non-technical syntax).
Regards,